test/gemm/gemm_kernel_base.hpp

```diff
@@ -56,13 +56,18 @@ namespace rocwmma
         virtual void          setup(ProblemParams const& problem)                 = 0;
         virtual void          exec()                                              = 0;
         virtual void          validateResults()                                   = 0;
-        virtual void          reportResults()                                     = 0;
+        virtual void          reportResults(std::ostream& stream,
+                                            bool isFstream,
```

Comment:
This injects a dependency on Fstream knowledge.

Instead, make this a boolean to for omitting the header.